package.json

```diff
@@ -47,35 +56,113 @@
     "eslint-plugin-prettier": "^3.0.0",
     "eslint-plugin-react": "^7.7.0",
     "eslint-utils": "^1.3.1",
+    "file-loader": "^4.1.0",
     "gh-pages": "^1.2.0",
     "husky": "^1.1.3",
+    "jest": "^24.8.0",
     "jest-dom": "^3.0.0",
+    "jest-environment-jsdom-fourteen": "^0.1.0",
+    "jest-resolve": "^24.8.0",
+    "jest-watch-typeahead": "^0.3.1",
     "jscodeshift": "^0.5.0",
     "npm-run-all": "^4.1.3",
+    "object-assign": "^4.1.1",
+    "postcss-flexbugs-fixes": "^4.1.0",
+    "postcss-loader": "^3.0.0",
     "prettier-eslint": "^8.8.2",
+    "promise": "^8.0.3",
+    "raf": "^3.4.1",
     "react": "^16.7.0",
+    "react-app-polyfill": "^1.0.1",
+    "react-dev-utils": "^9.0.1",
     "react-dom": "^16.7.0",
-    "react-styleguidist": "^7.3.10",
+    "react-styleguidist": "^9.1.12",
     "react-testing-library": "^5.4.4",
-    "watch": "^1.0.2"
+    "style-loader": "^0.23.1",
+    "typescript": "^3.5.3",
+    "url-loader": "^2.1.0",
+    "watch": "^1.0.2",
+    "webpack": "^4.37.0"
   },
   "dependencies": {
+    "autoprefixer": "^9.6.1",
+    "case-sensitive-paths-webpack-plugin": "^2.2.0",
     "focus-visible": "^4.1.5",
+    "fromentries": "^1.1.0",
     "react-color": "^2.17.0",
     "react-datepicker": "^2.3.0",
     "react-docgen-displayname-handler": "^2.1.1",
+    "react-icons": "^3.7.0",
```

Comment:
Useful?

Reply:
Yep. Actually we didn't know how it worked before without it. I think the previous webpack config was requiring it under the hood.